hw/ip/spi_device/lint/spi_device.waiver

```diff
@@ -122,6 +122,15 @@ waive -rules CASE_INC -location {spi_fwm_*xf_ctrl.sv} -regexp {Case statement ta
 waive -rules {NOT_USED NOT_READ} -location {spi_device.sv} \
       -regexp {'sub_(sram|p2s)_.*\[1\]' is not (used|read)} \
       -comment "CmdParse does not have SRAM intf"
+waive -rules {INPUT_NOT_READ HIER_BRANCH_NOT_READ} -location {spi_cmdparse.sv} \
+      -regexp {'cmd_info_i\[0:15\].*' is not read} \
+      -comment "Cmdparse uses opcode only"
+waive -rules {INPUT_NOT_READ HIER_BRANCH_NOT_READ} -location {spi_cmdparse.sv} \
+      -regexp {'cmd_info_i\[11:15\]\.opcode' is not read} \
+      -comment "Upper 5 slots are for passthrough only"
+waive -rules {INPUT_NOT_READ} -location {spi_cmdparse.sv} \
+      -regexp {'cmd_info_i\[15:11\]' is not read} \
+      -comment "Upper 5 slots are for the passthrough logic"
```

Comment:
nit: how about using `unused` assignments in RTL instead of adding a waiver for them?
I guess Verilator may complain about those in a similar way...

Reply:
OK. I wil add that too.

Reply:
Hum, tried it. This can be removed but other parts (one of struct memeber is not used) cannot be removed even I tied them into `unused_` signal.

Reply:
Hey Eunchan, thanks for adding the `unused_*` signals.
The reason that does not work is because of the AND masking to 0, since lint tools are smart enough to figure out that signals are not actually read in that case.
See proposed solution here: https://github.com/lowRISC/opentitan/pull/6707